package.json

```diff
@@ -34,7 +33,7 @@
     "@s-ui/studio": "8",
     "fs-extra": "8.1.0",
     "globby": "10.0.1",
-    "husky": "0.13.4",
+    "husky": "4.2.5",
```

Comment:
👀 that's quite a jump... was that intended?

Reply:
That's happening automatically when you perform a `npm install` as `sui-precommit` upgrades for you the husky dependency. Totally intended. :)

Reply:
This is because of:

> PR to improve sui-precommit:
> https://github.com/SUI-Components/sui/pull/896
>  Upgrade husky version to latest to fix problem that’s getting the default node version.
>  :new: feature: Now precommit is installed automatically on install.
>  :new: feature: Now on install migrate the old config to the new one.
>  :writing_hand: docs: some fixes to the README and documentation